src/Pages/DetailPage/Components/BedType.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const BedType = props => {
+  const { data } = props;
```

Comment:
함수형 컴포넌트에서는 이렇게 인자에서 바로 비구조화 하는 것도 있습니다!
```suggestion
const BedType = ({data}) => {
```

Reply:
넵~~